src/Table.cpp

```diff
@@ -666,25 +658,7 @@ namespace galsim {
                         (_vals[1] - _vals[0]) / (_args[1] - _args[0])) / (_args[2] - _args[0]);
 
         } else {
-            // For 4 or more points we use the TMV symmetric tridiagonal matrix solver
-
-#ifdef USE_TMV
-            tmv::SymBandMatrix<double> M(_n-2, 1);
-            for (int i=1; i<=_n-3; i++){
-                M(i, i-1) = _args[i+1] - _args[i];
-            }
-            tmv::Vector<double> rhs(_n-2);
-            for (int i=1; i<=_n-2; i++){
-                M(i-1, i-1) = 2. * (_args[i+1] - _args[i-1]);
-                rhs(i-1) = 6. * ( (_vals[i+1] - _vals[i]) / (_args[i+1] - _args[i]) -
-                                  (_vals[i] - _vals[i-1]) / (_args[i] - _args[i-1]) );
-            }
-            tmv::Vector<double> solution(_n-2);
-            solution = rhs / M;   // solve the tridiagonal system of equations
-            for (int i=1; i<=_n-2; i++) {
-                _y2[i] = solution[i-1];
-            }
-#else
+            // For 4 or more points we used to use the TMV symmetric tridiagonal matrix solver.
```

Comment:
Should this say `Eigen`?

Reply:
No, I think TMV.  We used to use TMV for this, which implements band matrices.  I didn't check again if Eigen has added them yet, but when I converted to 2.0 and added Eigen then, they didn't have any band matrix functionality.  So I implemented the version here.  But the historical reference is correct to say TMV.

Reply:
Ah, right.  I didn't read the comment closely enough.  I thought it said we were currently using TMV.